src/utils.js

```diff
@@ -162,6 +169,17 @@ export function projectKeyToPublicId(projectKey) {
   return keyToPublicId(projectKey)
 }
 
+/**
+ * Generate an invite ID from a project key
+ * @param {Readonly<Buffer>} projectKey
+ * @returns {Buffer}
+ */
+export function projectKeyToProjectInviteId(projectKey) {
```

Comment:
I think this belongs in @mapeo/crypto, so that we can version it separately. We can move it in a follow-up PR.

Reply:
Done in <https://github.com/digidem/mapeo-crypto/pull/26>.